access-token-management/samples/WebJarJwt/ClientAssertionService.cs

```diff
@@ -59,6 +59,11 @@ public ClientAssertionService(
                 { JwtClaimTypes.JwtId, Guid.NewGuid().ToString() },
                 { JwtClaimTypes.Subject, config.ClientId! },
                 { JwtClaimTypes.IssuedAt, DateTime.UtcNow.ToEpochTime() }
+            },
+
+            AdditionalHeaderClaims = new Dictionary<string, object>
+            {
+                { JwtClaimTypes.TokenType, "client-authentication+jwt" }
```

Comment:
```suggestion
                { JwtClaimTypes.TokenType, "client-assertion+jwt" }
```

@maartenba will merge if internal chat about this value is confirmed. Adding the suggestion here as a todo.